docs/issue-management.md

```diff
@@ -0,0 +1,395 @@
+`triage/unresolved` label and leave a reason in a comment for the original
+submitter. Unresolved issues can be closed after giving the original submitter
+an opportunity to appeal the reason supplied.
+
+### PR Triage
+
+A PR is triaged in the following way:
+
+1. Automation will ensure that the submitter has signed the [CLA](../CONTRIBUTING.md#cla).
+2. Automation will run CI tests against the submission to ensure compliance.
+3. Apply [`size/<size>`](#size) label to the submission. (TODO: we plan to
+   automate this with a GitHub action and apply size based on lines of code).
+4. Ensure that the PR references an existing issue (exceptions to this should be
+   rare). If the PR is missing this or needs any additional information, notate
+   in the comment and apply the `triage/needs-information` label.
+5. The PR should have the same `area/<area>`, `kind/<kind>`, and `lifecycle/<lifecycle>` labels as that of
+   the referenced issue. (TODO: we plan to automate this with a GitHub action
+   and apply labels automatically)
+
+## Working an Issue
+
+When starting work on an issue, assign the issue to yourself if it has not
+already been assigned and apply the `lifecycle/active` label to signal that the
+issue is actively being worked on.
+
+Making code changes is covered in detail in
+[../CONTRIBUTING.md](../CONTRIBUTING.md#github-workflow).
+
+If the issue kind is a `kind/bug`, ensure that the issue can be reproduced. If
+not, assign the `triage/not-reproducible` and request feedback from the original
+submitter.
+
+## Issue and PR Labels
+
+This section describes the label metadata we use to track issues and PRs. For a
+definitive list of all GitHub labels used within this project, please see
+[github-labels.md](github-labels.md).
+
+### Issue Kinds
+
+An issue kind describes the kind of contribution being requested or submitted.
+In some cases, the kind will also influence how the issue or PR is triaged and
+worked.
+
+#### API Change
+
+A `kind/api-change` label categorizes an issue or PR as related to adding, removing,
+or otherwise changing an API.
+
+All API changes must be reviewed by MAINTAINERS in addition to the standard code
+review and approval workflow.
+
+To create an API change issue or PR:
+
+* label your issue or PR with `kind/api-change`
+* describe in the issue or PR body which API you are changing, making sure to include
+  * API endpoint and schema (endpoint, Version, APIGroup, etc.)
+  * Is this a breaking change?
+  * Can new or older clients opt-in to this API?
+  * Is there a fallback? What are the implications of not supporting this API version?
+  * How is an upgrade handled? If automatically, we need to ensure proper tests
+    are created. If we require a manual upgrade procedure, this needs to be
+    noted so that the release notes and docs can be updated appropriately.
+
+Before starting any work on an API change it is important that you have proper
+review and approval from the project maintainers.
```

Comment:
nit: you use "MAINTAINERS" in some places and "maintainers" in others

Reply:
Will make this consistent.